### PR TITLE
Adds an Autodrobe to Selene dorms

### PR DESCRIPTION
**Pull request by ReinaCoder**
## About The Pull Request

Adds an Autodrobe to Selene maints. 

image
In game:
image

## Why It's Good For The Game

More clothing variety for people to wear during shifts.

## Changelog
:cl:
add: Adds an Autodrobe to Selene dorms
del: Removes a potted plant from Selene dorms
/:cl:

### CODE REVIEW

**Comment by SpaceDragon00** [comment]
> del: Removes a potted plant from Selene dorms

improve dont remove

File: _maps/map_files/SeleneStation/SeleneStation.dmm [hunk 1]
```diff
@@ -83743,13 +83743,13 @@
...
 /area/maintenance/solars/port/aft)
 "vTI" = (
-/obj/item/kirbyplants/random,
 /obj/effect/turf_decal/tile/neutral{
 	dir = 8
 	},
 /obj/effect/turf_decal/tile/neutral{
 	dir = 1
 	},
+/obj/machinery/vending/autodrobe,
```
Comment (Helianthus00) [inline _maps/map_files/SeleneStation/SeleneStation.dmm:83752]:
```suggestion
/obj/machinery/vending/autodrobe/all_access,
```
This is what you want instead

**Review by Helianthus00 — CHANGES_REQUESTED** [review]

File: _maps/map_files/SeleneStation/SeleneStation.dmm (hunk 1 above)
Comment (Helianthus00) [reply _maps/map_files/SeleneStation/SeleneStation.dmm:83752]: Also why is the branch name helio when this is a change on selene...

Comment (ReinaCoder) [reply _maps/map_files/SeleneStation/SeleneStation.dmm:83752]: Because helio and selene are the same maps

**Comment by ReinaCoder** [comment]
ok i think i changed it, but i dont know what those other two commits are

**Comment by Helianthus00** [comment]
You aren't updating your branch correctly. You need to rebase to fix all these extra files changed.

**Comment by ReinaCoder** [comment]
aaa

**Comment by ReinaCoder** [comment]
what the fuck

**Pull request closed without merging**